src/config/defaults.js

```diff
@@ -13,6 +13,7 @@ export default {
   STORE_DEVPORTAL_URL: 'https://dashboard.snapcraft.io/dev',
   STORE_ALLOWED_CHANNELS: ['edge'],
   STORE_PACKAGE_UPLOAD_REQUEST_LIFETIME: '7200',
+  BUILD_TRIGGER_THRESHOLD: 24,
```

Comment:
It would be worth adding it to the docs: https://github.com/canonical-websites/build.snapcraft.io/blob/master/docs/environment-variables.md